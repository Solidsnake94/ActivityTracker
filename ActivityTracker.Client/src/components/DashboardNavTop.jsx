import React, { Component } from "react";
import "../styles/Header.css";
import runningMan from "../images/running-man.svg";

class Header extends Component {
  render() {
    return (
      <header className="header">
        <h1 className="title">
          Activity Tracker - focus on running, leave the rest to us !
        </h1>
        <img src={runningMan} className="logo" alt="logo" />
      </header>
    );
  }
}

export default Header;
