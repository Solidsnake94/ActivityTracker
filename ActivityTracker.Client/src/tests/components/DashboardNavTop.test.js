import React from 'react';
import ReactShallowRender from 'react-test-renderer/shallow';

import DashboardNavTop from '../../components/DashboardNavTop';

test('should render dashboardnavtop correctly', () => {
    const renderer = new ReactShallowRender();

    renderer.render(< DashboardNavTop />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
})