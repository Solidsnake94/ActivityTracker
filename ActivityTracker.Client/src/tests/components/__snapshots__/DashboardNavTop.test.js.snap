// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboardnavtop correctly 1`] = `
<header
  className="header"
>
  <h1
    className="title"
  >
    Activity Tracker - focus on running, rest leave to us !
  </h1>
  <img
    alt="logo"
    className="logo"
    src="running-man.svg"
  />
</header>
`;
